nyc_taxis/track.py

```diff
@@ -0,0 +1,13 @@
+import time
+
+
+def wait_for_ml_lookback(es, params):
+    while True:
```

Comment:
Should we allow waiting for ever, if something goes wrong, or allow a timeout parameter as an escape hatch?

Reply:
I'd not do it at the moment because we usually have timeouts on other levels anyway.